pages/quasi-moon/index.vue

```diff
@@ -358,6 +384,32 @@ const goToPersonPage = (slug) => {
     }
   }
 
+  .about-card {
```

Comment:
i'd recommend not using the 80% width and using variable padding on the left and right: 
    padding-top: 1rem;
    padding-left: 8vh;
    padding-right: 8vh;